src/screens/contact/components/ContactMap.js

```
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+
+const ContactMap = ({ mapEmbed }) => (
+  <Card title="Bản đồ" bordered={false} style={{ minHeight: 300 }}>
+    {mapEmbed ? (
+      <div dangerouslySetInnerHTML={{ __html: mapEmbed }} style={{ width: '100%', minHeight: 250 }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)